AUTHORS.md

```diff
@@ -4,39 +4,39 @@
 ---
 # Authors
 
-* Alba Roza, [The Foundation for Public Code](https://publiccode.net/)
-* Anton Wiklund, [Swedish Public Employment Service](https://arbetsformedlingen.se/)
-* Arnout Engelen
+* Alba Roza, [The Foundation for Public Code](https://publiccode.net/) @AlbaRoza
+* Anton Wiklund, [Swedish Public Employment Service](https://arbetsformedlingen.se/) @ixuz
+* Arnout Engelen @raboof
 * Arnout Schuijff, The Foundation for Public Code
-* Audrey Tang, [digitalminister.tw](https://digitalminister.tw/)
-* Bastien Guerry, [DINUM](https://www.numerique.gouv.fr/dinum/)
-* Ben Cerveny, The Foundation for Public Code
+* Audrey Tang, [digitalminister.tw](https://digitalminister.tw/) @audreyt
+* Bastien Guerry, [DINUM](https://www.numerique.gouv.fr/dinum/) @bzg
+* Ben Cerveny, The Foundation for Public Code @nebster
 * Bert Spaan
-* Boris van Hoytema, The Foundation for Public Code
+* Boris van Hoytema, The Foundation for Public Code @bvhme
 * Charlotte Heikendorf
-* Claus Mullie, The Foundation for Public Code
+* Claus Mullie, The Foundation for Public Code @clausmullie
 * David Barberi
 * Edo Plantinga, [Code For NL](https://codefor.nl/)
-* Elena Findley-de Regt, The Foundation for Public Code
-* Eric Herman, The Foundation for Public Code
-* Felix Faassen, The Foundation for Public Code
+* Elena Findley*de Regt, The Foundation for Public Code @ElenaFdR
+* Eric Herman, The Foundation for Public Code @ericherman
+* Felix Faassen, The Foundation for Public Code @felixfaassen
 * Floris Deerenberg
```

Comment:
```suggestion
* Floris Deerenberg (@fdeerenberg)
```

Reply:
Yeah that's better.
I'll adjust the rest of the GitHub handles in a separate commit